src/pages/HomePage.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { Routes } from "../routes/router";
+
+function HomePage() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    navigate(Routes.SIGNUP);
```

Comment:
If you have a token stored you may want to redirect to dashboard instead of making the user go through the login again